xclim/sdba/nbutils.py

```diff
@@ -62,77 +71,65 @@ def vecquantiles(
 
 
 @njit
-def _quantile(arr, q):
-    if arr.ndim == 1:
-        out = np.empty((q.size,), dtype=arr.dtype)
-        out[:] = np.nanquantile(arr, q)
-    else:
-        out = np.empty((arr.shape[0], q.size), dtype=arr.dtype)
-        for index in range(out.shape[0]):
-            out[index] = np.nanquantile(arr[index], q)
+def _wrapper_quantile1d(arr, q):
+    out = np.empty((arr.shape[0], q.size), dtype=arr.dtype)
+    for index in range(out.shape[0]):
+        out[index] = _nan_quantile(arr[index], q)
```

Comment:
this is where numba has the best chance of helping, with the loop

Reply:
Ah yes indeed!